src/index.js

```diff
@@ -0,0 +1,52 @@
+var dotenv = require("dotenv");
```

Comment:
Do you need this? Usually you just pass things in as environment variables anyways. `.env` is a developer machine thing